domain-cc/cc-app/src/python_src/api.py

```diff
@@ -24,6 +24,14 @@
 )
 
 
+@app.get("/health")
+def get_health_status():
+    if not len(LOOKUP_TABLE):
+        raise HTTPException(status_code=500, detail="Lookup table is empty")
```

Comment:
Piggy backing on Ilya's comment... I'd say that the [503](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503) is better if the `LOOKUP_TABLE` is only temporarily down. If it going down means the service needs to be restarted or the `LOOKUP_TABLE` can't resume service, then a 500 might be better.

Reply:
@dfitchett i think to your point, it being down likely means that the service needs to be restarted so i think 500 might make sense here

Reply:
> HTTP 500 is usually reserved for failure that is unexpected. 503 would be a better choice.
> 
> Why should empty `LOOKUP_TABLE` cause to raise alert? What can cause `LOOKUP_TABLE` to be empty? This exception should probably be raised at that point. That would make troubleshooting much easier.

Re: 500 code, fair enough, ez fix. updated.


Re: Lookup table being empty.
It shouldn't be empty. Typically in a health check, I'd ping other resource like postgres, but we're really not doing much here, so just peeking at our in-memory mapping is the only thing I could consider to "check".

This would only happen if the .csv got wiped somehow. If we change the way that the lookup table is built (idk, maybe automating the build from the spreadsheet), this should cover that.

Reply:
It sounds that you put the if statement just to have something there.   It seems there is nothing actually special about `LOOKUP_TABLE` vs, some other object.  say... standard output.  The file that contains `printf` could magically disappear too.  IMHO it's better to have nothing than arbitrary code whose purpose is just to have something there.  (K.I.S.S / code is liability / etc) That would resolve the 500 vs 503 non-issue.  But I don't insist.  Up to you.